Cargo.toml

```
@@ -1,28 +1,36 @@
 [workspace]
 resolver = "2"
-members = ["demo"]
+members = ["examples/with_winit", "examples/run_wasm", "examples/scenes"]
 
 [workspace.package]
 edition = "2021"
-version = "0.0.1"
-license = "MIT OR Apache-2.0"
+version = "0.1.0"
+license = "Apache-2.0 OR MIT"
 repository = "https://github.com/linebender/velato"
 
 [package]
 name = "velato"
-description = "Lottie renderer built on vello."
+description = "A Lottie integration for vello."
 categories = ["rendering", "graphics"]
-keywords = ["2d", "vector-graphics", "animation", "lottie"]
-
+keywords = ["2d", "vector-graphics", "vello", "animation", "lottie"]
 version.workspace = true
 license.workspace = true
 edition.workspace = true
 repository.workspace = true
 
 [workspace.dependencies]
-vello = { git = "https://github.com/linebender/vello", rev = "b0303ccf98df15a8b196272720d364a56f7304ba" }
-
+# Update the README badges to match wgpu and vello version!
+vello = "0.1"
```

Let's specify the full version because otherwise `cargo upgrade` will ignore patch updates.

```suggestion
vello = "0.1.0"
```

---

This is interesting to hear... I thought by putting 0.1, anyone who uses Velato as a library can use any patch version, which gives maximum flexibility. Is this not what we want?

---

So, for someone using this library, there is absolutely zero difference between `0.1` and `0.1.0`. The difference is in relation to `cargo upgrade` (via [`cargo-edit`](https://crates.io/crates/cargo-edit)) which only modifies the numbers that exist. It's about maintenance for us, not about people actually using the library.

---

Is there ever a reason to use 0.1 or serde = "1"?

---

Reasons to not write the patch version (`0.1`) can include wanting to write fewer characters, easier reading, and signaling that you don't care about patch releases.

Serde `1` is wrong more often than not, because it makes a claim that `1.0.0` is sufficient which is often not the case with libraries as old as Serde. See [this thread](https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277) for one explanation.